### PR TITLE
Merge with Pharo (1). OSX 32/64 bits (3rd attempt)

### PR DESCRIPTION
**Pull request by estebanlm**
Well.. let's see how this goes. Message is same as other attempts: 

This is a first HUGE change :)
This change includes (not limited to):

- Correct Makefiles for building build.macos32x86/pharo.cog.spur
- Correct Makefiles for building build.macos64x64/pharo.cog.spur
- Merge back some platform files from Pharo branch (they should all be correctly delimited with PharoVM definition)

it is not finished, more changes will come

So, I hope this does not break anything... After this I will move to merge linux branches.

### CODE REVIEW

**Comment by estebanlm** [comment]
so... any observations with this? Can we accept it so I can move on?

File: platforms/iOS/vm/OSX/sqSqueakOSXOpenGLView.m [hunk 1]
```diff
@@ -477,72 +477,54 @@ - (void)doCommandBySelector:(SEL)aSelector {
...
+	else encode(  (isFunctionKey ? 4 : 29), (isFunctionKey ? 119 : 124), moveToRightEndOfLineAndModifySelection:)
+	else encode(  1, 115,   scrollToBeginningOfDocument:)
+	else encode(  4, 119,   scrollToEndOfDocument:)
+	else encode(  1, 115,   moveToBeginningOfDocumentAndModifySelection:)
+	else encode(  4, 119,   moveToEndOfDocumentAndModifySelection:)
+	else encode( 27, 53,    cancelOperation:)
+	else encode( 27, 53,    cancel:)
+	else encode( 27, 53,    complete:)
+	else encode( 27, 71,    delete:)
+	else return;
```
Comment (krono) [inline platforms/iOS/vm/OSX/sqSqueakOSXOpenGLView.m]: Oh thank you :D

**Comment by krono** [comment]
I'm good with this.

**Comment by estebanlm** [comment]
ok, no complain = merge! :)

**Pull request merged**